llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -2016,9 +2081,11 @@ TargetStackID::Value RISCVFrameLowering::getStackIDForScalableVectors() const {
 }
 
 // Synthesize the probe loop.
-static void emitStackProbeInline(MachineFunction &MF, MachineBasicBlock &MBB,
-                                 MachineBasicBlock::iterator MBBI,
-                                 DebugLoc DL) {
+MachineBasicBlock *RISCVFrameLowering::emitStackProbeInline(
```

Comment:
Why did the return type change? The caller doesn't use the returned MachineBasicBlock*

Reply:
I mixed up the patches again, this will only be needed for the dynamic allocation probing.
Removed.